### ISSUE TITLE
Encountered NPE when querying with BigtableSession

### ISSUE THREAD

**Issue by ankurcha** [GoogleCloudPlatform/cloud-bigtable-client#1492]
Found this error to occur consistently across our deployment using `bigtable-client-core:1.0.0-pre3`. 

```
[bigtable-aapi-grpc-555185655-dfkfv] E0828 18:09:54.280    grpc-default-executor-1609 SerializingExecutor.java:105] Exception while executing runnable io.grpc.internal.ClientCallImpl$1ClosedByContext@225df5e9 
[bigtable-aapi-grpc-555185655-dfkfv] java.lang.NullPointerException: null 
[bigtable-aapi-grpc-555185655-dfkfv] at com.google.cloud.bigtable.grpc.scanner.RetryingReadRowsOperation.onClose(RetryingReadRowsOperation.java:198) 
[bigtable-aapi-grpc-555185655-dfkfv] at com.google.cloud.bigtable.grpc.io.ChannelPool$InstrumentedChannel$2.onClose(ChannelPool.java:210) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:41) 
[bigtable-aapi-grpc-555185655-dfkfv] at com.google.cloud.bigtable.grpc.io.RefreshingOAuth2CredentialsInterceptor$UnAuthResponseListener.onClose(RefreshingOAuth2CredentialsInterceptor.java:416) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:419) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.internal.ClientCallImpl.access$100(ClientCallImpl.java:60) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.internal.ClientCallImpl$1ClosedByContext.runInContext(ClientCallImpl.java:160) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37) 
[bigtable-aapi-grpc-555185655-dfkfv] at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:102) 
[bigtable-aapi-grpc-555185655-dfkfv] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) 
[bigtable-aapi-grpc-555185655-dfkfv] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) 
[bigtable-aapi-grpc-555185655-dfkfv] at java.lang.Thread.run(Thread.java:748) 
```

I don't know if this is a corner case but looks like description of the status is empty that causes a npe.

**Comment by sduskis** [comment]
I just released a `1.0.0-pre4-SNAPSHOT` with this change.

**Comment by ankurcha** [comment]
Thanks!
